dist/index.js

```diff
@@ -83,16 +82,30 @@ class Product {
     }
 }
 const container = document.getElementById("contain");
+//built in products
 const product1 = new Product("onion", 5, 2, true, "Osem");
 const product2 = new Product("tomato", 3, 2, true, "Elite");
 const product3 = new Product("Snickers", 8, 2.5, false, "Usa");
 const product4 = new Product("Banana", 2, 1.6, true, "Elite");
-container.innerText += product1.returnHigherPrice(product1, product2);
+container.innerText += returnHigherPrice(product1, product2);
 container.innerHTML += "<br></br>";
-container.innerText += product1.returnHigherPrice(product2, product4);
+container.innerText += returnHigherPrice(product2, product4);
 container.innerHTML += "<br></br>";
 container.innerText += product1.expirationDate(product1, product2);
 product1.printProductToTable(product1);
-product1.printProductToTable(product2);
-product1.printProductToTable(product3);
-product1.printProductToTable(product4);
+product2.printProductToTable(product2);
+product3.printProductToTable(product3);
```

Comment:
this is not good
where is the usage in functions? generateUi methods? 
every thing is line by line?